addons/modules/functions/fnc_moduleSitOnChair.sqf

```diff
@@ -45,6 +51,8 @@
     180, \
     0, \
     180, \
+    0, \
```

Comment:
Adding 4 chair types but only 2 directions.

Reply:
Fixed it. Sorry about that.